api/ceramic.yaml

```diff
@@ -254,6 +254,21 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  "/network":
```

Comment:
Should we maybe nest this under some other namespace?  I'm still trying to think ahead to where we might have dozens of different things we want to be able to ask a node about its configuration.

`/config/network`?
`/nodeInfo/network`?

something like that?

Reply:
Yeah, I moved it under /config. I considered /node as that is similar to what js-ceramic used for things like chain info but went with config for now. Will merge tomorrow if no objections.